content/docs/pipelines/metrics/output-viewer.md

```diff
@@ -130,44 +132,55 @@ array:
       </tr>
       <tr>
         <td><code>source</code></td>
-        <td>The full path to the data. This path can contain wildcards ‘*’, in 
+        <td><p>The full path to the data. The available locations
+          include the local filesystem, <code>http</code>, <code>https</code>, 
```

Comment:
Local file system is not currently supported for output artifacts

Reply:
To clarify a little, minio offers users something akin to a local file system, but it's likely best to just list minio as an option, as you have already done

Reply:
Ah got it thanks. I'd seen the locations that looked like local files in the examples and was attempting to cater for those. Updated now.